content/scout/scores.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Docker Scout health scores
+description: |
+  Docker Scout health scores provide a supply chain assessment for Docker Hub
+  images, grading them from A to F based on various security policies.
+keywords: scout, health scores, evaluation, checks, grades, docker hub
+sitemap: false
+---
+
+> **Early Access**
+>
+> Health scores is an [Early Access](/release-lifecycle/#early-access-ea)
+> feature of Docker Scout. The feature is only available to organizations
+> participating in the early access program for this feature.
+{ .restricted }
+
+Docker Scout health scores provide a security assessment, and overall supply
+chain health, of images on Docker Hub, helping you determine whether an image
+meets your security standards. The scores range from A to F, where A represents
+the highest level of security and F the lowest, offering an at-a-glance view of
+the security posture of your images.
+
+## Scoring system
+
+Health scores are determined by evaluating images against a set of Docker Scout
+[policies](./policy/_index.md). Each policy is assigned a points value. If the
```

Comment:
Add this sentence here: ...a set of Docker Scout policies. **These policies align with best practices for the software supply chain and are recommended by Docker as foundational standards for images.**